concepts/eventing-code-guidelines.md

```diff
@@ -0,0 +1,1147 @@
+# Eventing team code guidelines
+
+**Document Intention** 
+
+The intention of this document is to provide style, testing, and code guidelines for components of the **eventing** team. Other teams in Kyma may follow different guidelines.
+The document is supposed to be a **living** document that can be changed by anyone in the team. However the team majority (50%) must agree on changes for this guideline. 
+
+This guide shall be publicly available so that it can be referenced when reviewing pull requests. The agreements of this guide shall not affect **external collaborators**. If an external PR does not align with our guidelines, we should accept the PR as it is and apply the guidelines ourselves in a follow-up PR so we don't block external contributors. Of course, also external PRs shall meet our quality standards but we shouldn't be too nitpicky.
+
+For now, the guidelines shall be applied to new code (PRs). There is no need to rewrite our entire code base now. This will gradually happen over time. As soon as the code reflects our guidelines, the need to look into this guide will decrease more and more.
+
+## Table of contents
+
+<!-- generate me with markdown-toc 
+```bash
+// source: https://github.com/jonschlinkert/markdown-toc
+markdown-toc -i --maxdepth 2 eventing-code-guidelines.md
+Do NOT TOUCH anything between the toc comments because this is used as a `marker` where to place the toc for markdown-toc.
+```
+-->
+
+<!-- toc -->
+
+<!-- tocstop -->
+
+## Recommended libraries
+
+The following section describes the **desired choice** of **libraries** for testing, logging, etc.
```

Comment:
no real need to highlight this at all, it's also the headline, thus emphasized by definition.

Reply:
I see your point 👍